README.md

```diff
@@ -17,8 +17,8 @@ editor.
 
 # AUTHORS
 
-*   Abdeali J Kothari - <mailto:abdealikothari@gmail.com>
-*   Lasse Schuirmann - <mailto:lasse.schuirmann@gmail.com>
+- Abdeali J Kothari - <mailto:abdealikothari@gmail.com>
```

Comment:
yep the problem here is that it's broken in one commit and fixed in the other, gitmate doesn't want you to break it in the first place basically